command/rootchain/fund/result.go

```diff
@@ -11,14 +11,16 @@ import (
 type result struct {
 	ValidatorAddr types.Address `json:"address"`
 	TxHash        types.Hash    `json:"tx_hash"`
+	IsMinted      bool          `json:"mint"`
```

Comment:
Do we even need `IsMinted`? If both fund and mint transactions are executed succesfully, `IsMinted` is always going to be true (otherwise command is not executed succesfully and results in an error).